spec/system/page_objects/pages/category.rb

```diff
@@ -65,6 +65,14 @@ def select_form_template(template_name)
         find(".select-category-template").click
       end
 
+      def find_new_topic_button
```

Comment:
This is pretty nit-picky, but I think it would be slightly better to only have one method here - 
`def new_topic_button` returns the button, and then in your spec you can do 
```
category_page.new_topic_button.click
```